nuget/IKVM.nuspec.in

```diff
@@ -5,9 +5,9 @@
     <authors>Jeroen Frijters</authors>
     <owners />
     <licenseUrl>http://sourceforge.net/apps/mediawiki/ikvm/index.php?title=License</licenseUrl>
-    <projectUrl>http://www.ikvm.net/</projectUrl>
-    <id>IKVM</id>
-    <title>IKVM.NET</title>
+    <projectUrl>https://github.com/wwrd/ikvm8</projectUrl>
```

Comment:
Do we update things like this now? Or merge and update after? (I'd say we should update the PR before mering)

Reply:
Good question, especially because it has already been mentioned that the [name might be changed](https://github.com/wwrd/ikvm8/issues/3#issuecomment-588359214). Not sure if we should discuss things likes names too extensively right now, can be changed later if needed as well. I've committed an example with changed links and names, not sure what exactly to change to what you had in mind.